infrastructure-formula/infrastructure/salt/master.sls

```diff
@@ -0,0 +1,42 @@
+{%- set extrascriptdir = '/usr/local/sbin/' -%}
+{%- set extrascripts = ['state-apply-super-async.sh'] -%}
+{%- set extrapackages = ['salt-keydiff'] + salt['pillar.get']('infrastructure:salt:formulas', []) -%}
+
+include:
+  - salt.master
+
+salt_master_extra_scripts:
+  file.managed:
+    - names:
+      - {{ extrascriptdir }}saltmaster-deploy:
+        - source: salt://infrastructure/salt/files{{ extrascriptdir }}saltmaster-deploy.j2
+        - template: jinja
+        - mode: '0700'
+{%- for file in extrascripts %}
+      - {{ extrascriptdir }}{{ file }}:
+        - source: salt://infrastructure/salt/files{{ extrascriptdir }}{{ file }}
+        - mode: '0755'
+{%- endfor %}
+
+salt_master_extra_packages:
+  pkg.installed:
+    - names:
```

Comment:
shouldn't that be `- pkgs:`?

(Looking it up in the documentation - `names` also works, but has a few disadvantages (like doing separate zypper runs for each package), and `pkgs` is recommended.)

Reply:
Right, corrected - I remember us chatting about this.